src/js/disability-benefits/526EZ/pages/paymentInfo.js

```diff
@@ -57,81 +56,64 @@ const paymentInformationViewField = ({ formData }) => {
-        'ui:options': {
-          hideIf: formData => formData.directDeposit.accountType === 'NOBANK'
-        }
-      },
-      routingNumber: {
-        'ui:title': 'Routing number',
-        'ui:options': {
-          hideIf: formData => formData.directDeposit.accountType === 'NOBANK'
-        }
-      },
-      bankName: {
-        'ui:title': 'Bank name',
-        'ui:options': {
-          hideIf: formData => formData.directDeposit.accountType === 'NOBANK'
-        }
-      },
-      'view:noBank': {
-        'ui:description': (<p>The Department of Treasury requires all federal benefit payments be made by electronic funds transfer (EFT), also called direct deposit. If you don’t have a bank account, you must get your payment through Direct Express Debit MasterCard. To request a Direct Express Debit MasterCard you must apply at <a href="https://www.usdirectexpress.com">www.usdirectexpress.com</a> or by telephone at 1-800-333-1795. If you chose not to enroll, you must contact representatives handling waiver requests for the Department of Treasury at 1-888-224-2950. They’ll address any questions or concerns you may have and encourage your participation in EFT.</p>),
-        'ui:options': {
-          hideIf: formData => formData.directDeposit.accountType !== 'NOBANK'
-        }
-      },
+export const uiSchema = {
+  'ui:title': 'Payment information',
+  'ui:field': ReviewCardField,
+  'ui:options': {
+    viewComponent: paymentInformationViewField
+  },
+  directDeposit: {
+    accountType: {
+      'ui:title':
+      'Please tell us where we should deposit your disability payment.',
+      'ui:widget': 'radio',
+      'ui:options': {
+        hideTitle: true,
+        labels: accountLabels
+      }
+    },
+    accountNumber: {
+      'ui:title': 'Account number',
+      'ui:options': {
+        hideIf: formData => formData.directDeposit.accountType === 'NOBANK'
+      }
+    },
+    routingNumber: {
+      'ui:title': 'Routing number',
+      'ui:options': {
+        hideIf: formData => formData.directDeposit.accountType === 'NOBANK'
+      }
+    },
+    bankName: {
+      'ui:title': 'Bank name',
       'ui:options': {
-        hideTitle: true
+        hideIf: formData => formData.directDeposit.accountType === 'NOBANK'
       }
+    },
+    'view:noBank': {
+      'ui:description': (<p>The Department of Treasury requires all federal benefit payments be made by electronic funds transfer (EFT), also called direct deposit. If you don’t have a bank account, you must get your payment through Direct Express Debit MasterCard. To request a Direct Express Debit MasterCard you must apply at <a href="https://www.usdirectexpress.com">www.usdirectexpress.com</a> or by telephone at 1-800-333-1795. If you chose not to enroll, you must contact representatives handling waiver requests for the Department of Treasury at 1-888-224-2950. They’ll address any questions or concerns you may have and encourage your participation in EFT.</p>),
```

Comment:
We may want to check with @peggygannon whether the phone numbers here should be hyperlinked, it's not clear that there is a standard convention for presenting phone numbers. If so, we can address in a later PR.

Reply:
Yeah, it probably should be hyperlinked. Seems like we've been consistent doing it that way of late.